.gitignore

```diff
@@ -1,8 +1,12 @@
 /target/
 .direnv
 .envrc
-.idea
 # generated by nix build, nix-build
 result
 # generated by the insta crate
 *.pending-snap
+
+# Editor specific ignores
+
+# JetBrains IDEs
```

Comment:
If this comment cannot go on the same line as `.idea`, it might not be worth spending two lines on it. Perhaps it's better to just get rid of it and have

```
# Editor-specific ignores
.idea
```

Reply:
Yeah, that isn't valid syntax. OK, I'll remove the add'l lines.

Reply:
Thanks!